tools/travis-pull-db.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Environment variable DB is used by this script.
+# If DB is undefined, than this script does nothing.
```

Comment:
`than` should be `then` -> "If DB is undefined, then this script does nothing."

Reply:
dnoe :)